templates/Java/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -1,1885 +0,0 @@
-{{>licenseInfo}}
-
-package {{invokerPackage}};
-
-{{#dynamicOperations}}
-import io.swagger.v3.oas.models.OpenAPI;
-import io.swagger.v3.oas.models.Operation;
-import io.swagger.v3.oas.models.PathItem;
-import io.swagger.v3.oas.models.parameters.Parameter;
-import io.swagger.v3.oas.models.parameters.Parameter.StyleEnum;
-import io.swagger.v3.parser.OpenAPIV3Parser;
```

Comment:
There was a change in this template to modify the default timeouts returned by the API Client. You'll need to maintain that same default in the rest-assured API client.

Reply:
what "default timeouts" you mean?